README.md

```diff
@@ -4,68 +4,67 @@
 
-While Litmus experiments were developed initially to test if a given stateful workload is suitable for running 
+While the experiments were developed initially to test if a given stateful workload is suitable for running 
 on [OpenEBS](www.openebs.io)(_a Kubernetes dynamic storage provisioner_); the use cases are broader and overall 
 system resilience can be characterized before and during operations.  
 
-## How Litmus is different from others
-Litmus is an umbrella project that incorporates different pieces of a typical chaos engineering environment to deliver a 
-complete solution to its users. Some of the core components include: 
-
-- [Litmus](https://github.com/litmuschaos/litmus): The actual execution framework & repository of ready/configurable chaos 
-  experiments (mostly written as ansible playbooks & executed as Kubernetes jobs). The jobs are often executed in CI pipelines 
-  as part of e2e (refer https://openebs.ci) 
-
-- [Chaos-Operator](https://github.com/litmuschaos/chaos-operator): A Kubernetes Operator that watches & acts on custom 
-  resources defining litmus chaos experiment workflows. Typically used in deployment environments (dev/staging/pre-prod/prod) 
-  where chaos experiments can be scheduled & monitored against specific applications. 
+ The actual execution framework & repository of ready/configurable test experiments (mostly written as ansible playbooks & executed as Kubernetes jobs). The jobs are often executed in CI pipelines as part of e2e (refer https://openebs.ci) 
 
-- [Chaos-Exporter](https://github.com/litmuschaos/chaos-exporter): A Prometheus Exporter that exposes chaos metrics based 
-  on experiment results.
-
-The chaos experiments make use of facilitator containers in [test-tools](https://github.com/litmuschaos/test-tools) to 
+The test experiments make use of facilitator containers in [test-tools](https://github.com/litmuschaos/test-tools) to 
 implement the chaos, load generation, logging and other utility functions. 
 
-With Litmus, the test logic is packaged into dedicated containers which makes them portable across Kubernetes deployments. 
+The test logic is packaged into dedicated containers which makes them portable across Kubernetes deployments. 
 This containerization also helps to integrate Litmus into CI/CD environments. 
 
 And as a developer friendly framework, it also provides helpful playbooks to quickly spin-up Kubernetes clusters on different 
 cloud & on-premise platforms on which to run the experiments! 
 
-For details on the architecture, implementation & reference usecases, please read the [litmus docs](https://docs.litmuschaos.io)
-
 ## Getting Started
 
-Litmus experiment jobs(also called Litmusbooks) run using a dedicated ServiceAccount in the Litmus namespace. Setup RBAC & custom
-resource definitions (CRDs) via kubectl or helm, as shown below: 
+The test experiment jobs(also called Litmusbooks) run using a dedicated ServiceAccount in the Litmus namespace. Setup RBAC & custom
```

Comment:
`also called Litmusbooks` is this needed?

Reply:
yes